src/docs/eviction.dox

```diff
@@ -0,0 +1,102 @@
+/*! @m_page{{c,java},eviction, Eviction}
+
+All the operations performed in WiredTiger are on the data read into
+memory. WiredTiger uses memory as a cache for all the data on the disk
+and the data in memory forms the current working set.
+
+Since memory is limited and can't hold all the data stored on the disk,
+WiredTiger has to continuously move data that is currently not being
+accessed out of the memory to free up enough space to read data that
+are requested by the user but currently reside on the disk back into
```

Comment:
The word data is a plural noun so write "data are". Datum is the singular.

Reply:
It is grammatically acceptable to use the singular when treating 'data' as a mass noun. Since you are treating data as plural, however, you want to change the beginning of this sentence from "data that is" to "data that are".